lib/commands/instrument.js

```diff
@@ -28,6 +29,10 @@ exports.builder = function (yargs) {
       type: 'boolean',
       description: "should nyc's instrumenter produce source maps?"
     })
+    .option('cwd', {
```

Comment:
`cwd` is actually already defined in `lib/config-util.js` in a way that allows it to be global.  I think we should not add the `.option` to this file but we should update the `describe` string in `lib/config-util.js`.  Note the current definition of `lib/config-util.js` often does not result in `cwd: process.cwd()`.  It uses `find-up` to search for the `package.json` which owns `process.cwd()`, then uses the dirname of that `package.json`.

Existing: `working directory used when resolving paths`
Maybe change to: `base directory of this project`

The important thing is that `--cwd` effects the interpretation of `include` and `exclude`.  I think it should not effect the interpretation of the input/output positional arguments.  Consider this, when you run `nyc --cwd=.. ./somescript.js` the `./somescript.js` is found from $PWD, not from the value of `--cwd`.